website/docs/d/synthetics_locations.html.markdown

```diff
@@ -0,0 +1,18 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_synthetics_locations"
+sidebar_current: "docs-datadog-datasource-synthetics-locations"
+description: |-
+  Get information on Datadog's Synthetics locations for Synthetics tests
+---
+
+# datadog_synthetics_locations
+
+Use this data source to retrieve information about Datadog's Synthetics Locations.
+## Example Usage
+
+```hcl
+data "datadog_synthetics_locations" "test" {}
+```
+
+## Attributes Reference
```

Comment:
This should include a description for `locations` right?

Reply:
Yes indeed, will update